### PR TITLE
fix: replace xml parser

### PR DESCRIPTION
**Pull request by doouding**

### CODE REVIEW

**Comment by doouding** [comment]
* **#8267** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8267?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @doouding and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by doodlewind — APPROVED** [review]

File: packages/blocks/package.json [hunk 1]
```diff
@@ -63,9 +63,9 @@
     "remark-parse": "^11.0.0",
     "remark-stringify": "^11.0.0",
     "shiki": "^1.14.1",
+    "simple-xml-to-json": "^1.2.2",
```
Comment (Saul-Mirone) [inline packages/blocks/package.json:66]: I personally recommend https://lezer.codemirror.net/

Comment (Saul-Mirone) [reply packages/blocks/package.json:66]: https://github.com/lezer-parser/xml

Comment (doouding) [reply packages/blocks/package.json:66]: It may take some time to adopt this module. I'll come back later.

**Pull request merged**